README.md

```diff
@@ -97,6 +97,13 @@ Follow this guide to setup your development machine.
 
     ```
     npm install
+    ``` 
+    If you don't have bower installed global, run the following:
```

Comment:
Can you please change the text to -> `Install bower(1.8.0) globally by running:`
 `npm install -g bower`